.github/workflows/pypi_publish.yml

```
@@ -59,3 +129,209 @@ jobs:
+      - name: Post success to Slack channel dev-connectors-extensibility
+        uses: slackapi/slack-github-action@v1.23.0
+        # TODO: Remove the 1 == 0 condition when we're ready to post to Slack
+        if: 1 == 0
+        continue-on-error: true
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN_AIRBYTE_TEAM }}
+        with:
+          channel-id: C04J1M66D8B
+          # Channel: #dev-connectors-extensibility-releases
+          # Link (internal): https://airbytehq-team.slack.com/archives/C04J1M66D8B
+          payload: |
```

actionlint (1.7.4)

property "changelog-message" is not defined in object type {publish_to_dockerhub: string; publish_to_pypi: string; update_connector_builder: string; version: string}

(expression)